README.md

```diff
@@ -73,10 +139,23 @@ We thank the following people for their extensive assistance in the development
 
 If you would like to contribute to this pipeline, please see the [contributing guidelines](.github/CONTRIBUTING.md).
 
-## Citations
+Resources
+-----
+The documentation of the various tools used in this workflow are available here:
+
+[Nextflow](https://www.nextflow.io/docs/latest/index.html)
+
+[bcftools](https://samtools.github.io/bcftools/bcftools.html)
 
-<!-- TODO nf-core: Add citation for pipeline after first release. Uncomment lines below and update Zenodo doi and badge at the top of this file. -->
-<!-- If you use ferlab/postprocessing for your analysis, please cite it using the following doi: [10.5281/zenodo.XXXXXX](https://doi.org/10.5281/zenodo.XXXXXX) -->
+**GATK**:
+- [CombineGVCFs](https://gatk.broadinstitute.org/hc/en-us/articles/360037593911-CombineGVCFs)
+- [GenotypeGVCFs](https://gatk.broadinstitute.org/hc/en-us/articles/360037057852-GenotypeGVCFs)
+- [VariantRecalibrator](https://gatk.broadinstitute.org/hc/en-us/articles/360035531612-Variant-Quality-Score-Recalibration-VQSR)
+- [VariantFiltration](https://gatk.broadinstitute.org/hc/enus/articles/360041850471-VariantFiltration))
+
+[VEP](https://useast.ensembl.org/info/docs/tools/vep/script/vep_options.html)
+
```

Comment:
Do you want to add a note about the human genome used?

Reply:
I think this pipeline will give us a lot more flexibility concerning the genomes, because it links to the relevant files from different versions of the human genome from igenome , i.e. with the "genome" parameter. I will add some info about it